Example/Example/AnimationListView.swift

```diff
@@ -0,0 +1,93 @@
+// Created by Cal Stephens on 7/11/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import Lottie
+import SwiftUI
+
+// MARK: - AnimationListView
+
+struct AnimationListView: View {
+
+  let directory: String
+
+  var body: some View {
+    List {
+      ForEach(items, id: \.self) { item in
+        NavigationLink(value: item) {
```

Comment:
over here googling simple things like "swiftui how to push on navigation stack"

Reply:
I have now learned that there are 5-6 different APIs for doing this and I don't know the differences. Classic SwiftUI.